snow/networking/handler/handler.go

```diff
@@ -423,6 +423,8 @@ func (h *handler) handleSyncMsg(ctx context.Context, msg Message) error {
 		op        = msg.Op()
 		body      = msg.Message()
 		startTime = h.clock.Time()
+		// Check if the chain is in normal operation at the start of message execution (may change during execution)
+		isNormalOp = h.ctx.State.Get().State == snow.NormalOp
```

Comment:
Per @StephenButtolph 's suggestion, we should care about whether the operation is in normal operation at the start of processing instead of the end.

If the state changes during the execution of the message, then we would still expect that this message may take a much longer time to process since the blockchain is still bootstrapping.